modules/nf-core/coptr/map/meta.yml

```diff
@@ -0,0 +1,63 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "coptr_map"
+description: write your description here
+keywords:
+  - coptr
+  - mapping
+  - ptr
+tools:
+  - "coptr":
+      description: "Accurate and robust inference of microbial growth dynamics from
+        metagenomic sequencing reads."
+      homepage: "https://github.com/tyjo/coptr"
+      documentation: "https://coptr.readthedocs.io/"
+      tool_dev_url: "https://github.com/tyjo/coptr"
+      doi: "10.1101/gr.275533.121"
+      licence: ["GPL v3"]
+      identifier: biotools:coptr
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - fasta:
+        type: file
+        description: fastq file with reads
+        pattern: "*.{.fastq,.fq,.fastq.gz,fq.gz}"
+        ontologies:
+          - edam: "http://edamontology.org/format_1929"
+  - - meta2:
+        type: map
+        description: |
+          Groovy Map containing index genome id and path
+          e.g. [ id:'test', 'bowtie2' ]
+    - index:
+        type: file
+        description: Directory with Bowtie2 genome index files
+        pattern: "*.ebwt"
+
+output:
+  - bam:
+      - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*.bam":
+          type: file
+          description: BAM file
```

Comment:
I imagine what that BAM might contain, but please make it a bit more explicit just in case